test/recipe/admin/components/Baked/Menu.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <PanelMenu v-if="schema" :model="items" class="w-full md:w-80">
```

Comment:
i think in order to make this full width, we need to have an extra layout feature in record component

- either to disable grid
- or to have a full width title section before props